portfolio/src/main/webapp/index.html

```diff
@@ -3,17 +3,210 @@
+            </div>
+            
+            <div class="col-sm form-div">
+              <form class="form-style" action="https://formspree.io/f/xyynkveg" method="POST">
+                <ul>
+                  <div>
+                    <li>
+                      <input type="text" id="name" name="name" class="input-style" placeholder="name">
+                    </li>
+                    <li>
+                      <input type="text" id="email" name="_replyto" class="input-style" placeholder="email">
+                    </li>
+                    <li>
+                      <textarea type="text" id="message" name="message" placeholder="message"></textarea>
+                    </li>
+                    <li class="button">
+                      <button type="submit">send</button>
+                    </li>
+                  </div>
+                  
+                </ul>
+              </form>
+
+            </div>
+      
+          </div>
+          
+      </section>
+    </main>
+
+
+    <footer>
+      <div class="container">
+        <div class="social-icons-container">
+          <a href="https://medium.com/me/stories/public" target="_blank"><img src="img/medium.svg" class="social-icon"/></a>
+          <a href="https://github.com/nblake707" target="_blank"><img src="img/github.svg" class="social-icon"/></a>
+          <a href="https://www.linkedin.com/in/nateyanablake/" target="_blank"><img src="img/linkedin.svg" class="social-icon"/></a>
+          <a href="https://open.spotify.com/user/nblake2?si=ZF3FY469SWe-4l-xjt7JEQ" target="_blank"><img src="img/spotify.svg" class="social-icon"/></a>
+      </div>
+      <p>Made with love in NYC</p>
+    </div>
+    </footer>
+
+    <!--  JS  -->
+
+    <!-- must be loaded first for external js file to work -->
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    
+    
+    
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script> 
+   <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.min.js" integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF" crossorigin="anonymous"></script>
+    
+    
+    <script src="https://cdn.jsdelivr.net/npm/kursor"></script>
+    
+
+
+    <script>
+      new kursor({
+          type: 2,
+          removeDefaultCursor: true,
+          color: '#273E47',
+      })
+  </script>
```

Comment:
Can you add a comment on what this block of code does?

Reply:
This is a library I'm using that changes the appearance of the cursor.